examples/todo-jwt/CHANGELOG.md

```diff
@@ -0,0 +1,1509 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
wouldn't this file generate automatically?

Reply:
will remove. thanks.